workspace/utilities/com/image.xsl

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: image =====================================================================-->
+	<xsl:template name="image">
+		<xsl:param name="image" select="image" />
+		<xsl:param name="alt" select="image-alt" />
+		<xsl:param name="veil-opacity" select="veil-opacity" />
+		<xsl:param name="is-cover" select="false()" />
+		<xsl:param name="is-contain" select="false()" />
```

Comment:
True! I would still only implement resize, contain & cover in the template to avoid bloating the file since they are the one used in 99% of cases. What do you think?

Reply:
I agree with you, one thing that would be nice is the render image without resize; or a way to remove the JIT functionality easily.

Reply:
It's a nice option @fhamon. I don't see how without doubling the chooses right now so I suggest adding it at a later time. Or if someone has an idea, please come forward ;)